include/guestbook.theme.inc

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @file
+ * The implementation of the theme functions.
+ */
+
+/**
+ * Generates messages html markup.
+ *
+ * @return
+ *   Html markup.
+ */
+function guestbook_html() {
+  global $user;
+
+  // Variable $mes is fully generated html markup.
+  $mes = NULL;
+
+  // Get messages data array from database.
+  $variables = guestbook_read_db('guestbook_drupal', 'id', 'DESC', '10');
```

Comment:
these data should be passed to theme callback, do not use here connection to db and make some queries